docs/src/flavors/clusterclass-kubeadm.md

```diff
@@ -0,0 +1,59 @@
+# Kubeadm ClusterClass
+## Specification
+| Control Plane | CNI    | OS     | Installs ClusterClass |
+|---------------|--------|--------|-----------------------|
+| Kubeadm       | Cilium | Ubuntu | Yes                   | 
+## Prerequisites
+[Quickstart](../topics/getting-started.md) completed
+## Usage
+### Create clusterClass and first cluster
+1. Generate clusterClass & cluster manifests
+    ```bash
+    clusterctl generate cluster test-cluster --flavor clusterclass-kubeadm > test-cluster.yaml
+    ```
+2. Apply cluster manifests
+    ```bash
+    kubectl apply -f test-cluster.yaml
+    ```
+### Create second cluster
```

Comment:
I would note that this is optional, e.g. "you can create a second cluster with the existing ClusterClass as follows"